t/24-worker-overall.t

```diff
@@ -538,6 +538,15 @@ subtest 'handle client status changes' => sub {
     like($output, qr/Test disabling - ignoring server/s, 'client disabled');
     unlike($output, qr/Stopping.*because registration/s, 'worker not stopped; there are still other clients');
 
+    # assume one of the clients is suffering from clock skew
+    $ENV{OPENQA_WORKER_CONNECT_INTERVAL} = 7;
+    $fake_client_2->status('failed');
+    combined_like {
+        $worker->_handle_client_status_changed($fake_client_2,
+            {status => 'failed', reason => 'test', error_message => '403 response: timestamp mismatch'})
+    }
+    qr/trying again in 7 seconds/s, 'worker will attempt to retry later';
```

Comment:
Looks like we don't test this case so far, at least not with an explicit check for the log message. So I guess it is good to have.

Having that said, this test is *not* covering the new/additional branch. Not sure whether that's necessary, though.

Reply:
You're right, I updated the test phrasing. This code is not exercising the registration case...